go-controller/hybrid-overlay/pkg/types/types.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	kapi "k8s.io/api/core/v1"
+)
+
+const (
+	// HybridOverlayAnnotationBase holds the hybrid overlay annotation base
+	HybridOverlayAnnotationBase = "k8s.ovn.org/hybrid-overlay-"
+	// HybridOverlayNodeSubnet holds the pod CIDR assigned to the node
+	HybridOverlayNodeSubnet = HybridOverlayAnnotationBase + "node-subnet"
+	// HybridOverlayDRMAC holds the MAC address of the Distributed Router/gateway
+	HybridOverlayDRMAC = HybridOverlayAnnotationBase + "distributed-router-gateway-mac"
+
+	// HybridOverlayVNI is the VNI for VXLAN tunnels between nodes/endpoints
+	HybridOverlayVNI = 4097
```

Comment:
I wonder if it would make sense to use this as a default and actually read it from a ConfigMap, to make it configurable by the user?
If not for now, maybe for a PR afterwards.

Reply:
@rcarillocruz yeah it does make sense to have it configurable in the future.